### ISSUE TITLE
Last song remembered at startup but not last playlist

### ISSUE THREAD

**Issue by fluxish** [Quacky2200/Spotify-Web-Player-for-Linux#69]
Hi, the web player, at startup, remembers the last song played but not the last playlist. Indeed, it plays other songs in the album of the last song.

**Comment by Quacky2200** [comment]
Yeah this is a common issue with Spotify Web Player itself than the application.
